### PR TITLE
MINOR Added 6.21. Use cookies/sessions securely

### PR DESCRIPTION
**Pull request by js-kyle**
I would like to contribute to this project, so I have added to the security section with some information around leaving session middleware to it's default config.

Let me know any feedback or where else I can contribute! Thanks

### CODE REVIEW

**Comment by BrunoScheufler** [comment]
@js-kyle thanks for the awesome contribution 🥇 I'll review the changes later, maybe @i0natan can have a look at it, too!

**Comment by BrunoScheufler** [comment]
@js-kyle I'd like to let @i0natan look over the changes, the content seems to fit the section, but we'd have to decide the order/position of the point as it has been changed in the [current security-best-practices-section branch](https://github.com/i0natan/nodebestpractices/tree/security-best-practices-section#6-security-best-practices).

File: README.md [hunk 1]
```diff
@@ -817,6 +817,12 @@ All statements above will return false if used with `===`
 
 🔗 [**Read More: Common security best practices**](/sections/security/commonsecuritybestpractices.md)
 
+## ![✔] 6.15. Use cookies/sessions securely
```
Comment (i0natan) [inline README.md]: Title should be related to Node.JS (not generic security advice). 

For example: Modify the default session middlewares settings

File: README.md [hunk 2]
```diff
@@ -817,6 +817,12 @@ All statements above will return false if used with `===`
...
+
+**TL;DR:** Using the default settings for session middleware can be expose your app to module and framework specific hijacking attacks in a similar way to the  `X-Powered-By` header.
+
+
```
Comment (i0natan) [inline README.md]: Missing 'Otherwise' section here (helps the reader understand the advice impact)

**Comment by i0natan** [comment]
@js-kyle welcome aboard! 

I think that this bullet is helpful as it emphasizes a node-related issue and not just a generic security advice. I would just change the title to be Node-related (see my inline comment with suggestion), add 'Otherwise' section.

We'd love to add you to our collaboration on this section and in general to work together.

**Comment by js-kyle** [comment]
@i0natan That makes sense - I've added the Node specific title and added the otherwise section.

Sounds good!

**Comment by i0natan** [comment]
@js-kyle would you like to join us in writing the other bullets?

@BrunoScheufler I believe that @js-kyle deserve a star (approved new bp), no? :)

**Comment by BrunoScheufler** [comment]
@i0natan quite certainly, he does! The PR should although be with an updated version of the branch because the bullet number (6.15) seems incorrect as of now

**Comment by js-kyle** [comment]
@i0natan Yes, very keen to pick up some more bullets.

@BrunoScheufler I'll get this PR updated with appropriate bullet number 👍

**Comment by js-kyle** [comment]
@BrunoScheufler - this is now bullet 6.21

**Comment by BrunoScheufler** [comment]
@js-kyle awesome! :tada:

**Pull request merged**